File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/20/20.js
// Q20: Print number triangle (1, 12, 123, 1234, 12345).
// Input: An integer n
// Output: Number triangle pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 1; i <= n; i++) {
        let row = "";
        for (let j = 1; j <= i; j++) {
            row += j;
        }
        console.log(row);
    }
    rl.close();
});

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/20/20.py
# Q20: Print number triangle (1, 12, 123, 1234, 12345).
# Input: An integer n
# Output: Number triangle pattern

n = int(input())
for i in range(1, n + 1):
    for j in range(1, i + 1):
        print(j, end="")
    print()
